packages/app-mobile/utils/fs-driver/fs-driver-rn.ts

```
@@ -18,24 +18,49 @@ function isScopedUri(path: string) {
 	return path.includes(ANDROID_URI_PREFIX);
 }
 
+// Converts some encodings specifiers that work with NodeJS into encodings
+// that work with RNSAF, RNFetchBlob.fs, and RNFS.
+const normalizeEncoding = (encoding: string) => {
+	encoding = encoding.toLowerCase();
+
+	// rn-fetch-blob and RNSAF require the exact string "utf8", but NodeJS (and thus
+	// fs-driver-node) support variants on this like "UtF-8" and "utf-8". Convert them:
+	if (encoding === 'utf-8') {
+		encoding = 'utf8';
+	}
+
+	// Warn if unable to convert to a supported encoding.
+	const supportedEncodings = ['utf8', 'ascii', 'base64'];
+	if (!supportedEncodings.includes(encoding)) {
+		logger.warn(`Unsupported encoding: ${encoding}.`);
```

I think we could throw an error here, as if we proceed we might write invalid data